dts/arm/atmel/same70.dtsi

```diff
@@ -419,7 +419,8 @@
 
 		can0: can@40030000 {
 			compatible = "atmel,sam-can";
-			reg = <0x40030000 0x100>;
+			reg = <0x40030000 0x100>, <0x40088110 0x01>;
+			reg-names = "m_can", "dma_base_reg";
```

Comment:
- A length of `0x01` is wrong. The length is specified in bytes.
- Regarding the naming, please choose a name as close to the naming used in the SoC data sheet/reference manual.
- Please update the binding to describe the new, required `reg` entry.
- Please update the release notes as well.

Reply:
> * A length of `0x01` is wrong. The length is specified in bytes.
> * Regarding the naming, please choose a name as close to the naming used in the SoC data sheet/reference manual.
> * Please update the binding to describe the new, required `reg` entry.

done

Reply:
> * Please update the release notes as well.

not sure about how to do that, could you please give me a hint?

Reply:
> * Please update the release notes as well.

just found it. done